src/components/manage_users_table/ManageUsersTableCellData.js

```diff
@@ -17,51 +34,51 @@ export function usernameCellData({displayName, userId, label= 'user-name'}) {
   };
 }
 
-export function googleIdCellData({userId, email, label = 'google-id'}) {
+export function emailCellData({userId, email, label = 'email'}) {
   return {
     data: email,
     value: email,
     id: userId,
     style: {
-      color: styles.color.googleId,
-      fontSize: styles.fontSize.googleId,
+      color: styles.color.email,
+      fontSize: styles.fontSize.email,
       fontWeight: '500'
     },
     label
   };
 }
 
-//Redirect for admin review page, only used in admin manage dar collections table
-export function roleCellData({userId, roles, label = 'user-role'}) {
+export function permissionsCellData({userId, roles, libraryCards, label = 'permissions'}) {
+  const hasLibraryCard = !isNil(libraryCards) && !isEmpty(libraryCards);
+  const roleNames = map(roles, 'name').filter((roleName) => roleName !== 'Researcher');
+  const perms = (hasLibraryCard ? roleNames.concat('LibraryCard') : roleNames);
+
+  // need to split, e.g., SigningOfficial -> Signing Official
+  // use regex to split by case, then stitch back together with spaces
+  // regex from: https://gist.github.com/JeffJacobson/3841577
+  const splitByWordRegex = /($[a-z])|[A-Z][^A-Z]+/g;
+  const formattedPerms = perms.map((perm) => perm.match(splitByWordRegex).join(' '));
+
   return {
     isComponent: true,
-    data: sortedUniq(map(roles, 'name')).join('   '),
+    data: sortedUniq(formattedPerms).join('   ') || 'None',
     label,
     id: userId,
   };
 }
 
-export function actionsCellData({userId, user, editUser}) {
+export function institutionCellData({userId, institution, label = 'insitution'}) {
   return {
     isComponent: true,
+    data: institution?.name || 'N/A',
```

Comment:
TDIL! This is something I [absolutely loved in groovy](https://groovy-lang.org/operators.html#_safe_navigation_operator) and did not know this was added to JS.